storage/src/freezer/storage.rs

```diff
@@ -1018,3 +1018,33 @@ impl<E: Storage + Metrics + Clock, K: Array, V: Codec> Freezer<E, K, V> {
         self.resizable
     }
 }
+
+impl<E: Storage + Metrics + Clock, K: Array, V: Codec> crate::store::Store for Freezer<E, K, V> {
+    type Key = K;
+    type Value = V;
+    type Error = Error;
+
+    async fn get(&self, key: &Self::Key) -> Result<Option<Self::Value>, Self::Error> {
+        self.get(Identifier::Key(key)).await
+    }
+}
+
+impl<E: Storage + Metrics + Clock, K: Array, V: Codec> crate::store::StoreMut for Freezer<E, K, V> {
```

Comment:
While you're here would you mind updating the function documentation for `put` clarifying what happens if you put the same key more than once?  Found it a bit surprising that this is allowed given the freezer is described as "immutable".

Reply:
Added comment on `put`: `/// If the key already exists, the value is updated.`